app/src/test/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetControllerTest.kt

```diff
@@ -307,9 +307,7 @@ class BloodPressureEntrySheetControllerTest {
     verify(ui).showConfirmRemoveBloodPressureDialog(bloodPressure.uuid)
   }
 
-  // TODO Migrate logic to Mobius
   @Test
-  @Suppress("IMPLICIT_CAST_TO_ANY")
```

Comment:
I'm curious, why is this no longer required? Was the problem fixed with either a Kotlin version update, or the Android SDK update?

Reply:
It was removed because we replaced the `List<Any>` with a `List<TestParams>`.